tuf/client/client.go

```diff
@@ -88,7 +88,7 @@ func (c *Client) downloadRoot() error {
 		logrus.Debugf("Loading root with no expected checksum")
 
 		// get the cached root, if it exists, just for version checking
-		cachedRoot, _ := c.cache.GetMeta(role, -1)
+		cachedRoot, _ := c.cache.GetSized(role, -1)
```

Comment:
I'm guessing when we get to that PR we will have to make that update but I don't want to do it preemptively.